components/src/core/list-item-tag/list-item-tag.component.scss

```diff
@@ -15,6 +15,7 @@
         font-weight: 700;
         text-transform: uppercase;
         letter-spacing: 1px;
+        margin-right: -1px;
```

Comment:
should letter spacing be specified in rems to look correct when the font is anything other than 16px?

Does the margin-right approach work when you flip to RTL with a true RTL language?

Reply:
I checked this with RTL language but the text still had the letter-spacing applied to the right side instead of the left, so it seems this margin adjustment works without RTL special attention.  

For larger font sizes, I have to check that out and see.

Reply:
Since the letter spacing is set to be 1px, the negative 1px adjustment works with all font sizes.

Reply:
But does the letter spacing itself look correct when we specify 1px? I.e., 1px between letters that are 16px tall looks a certain way, 1px between letters that are 40px tall might not look how we intend.

Reply:
If someone set their system font size to be x2.5 of our default, I would imagine a lot of designs to go pretty off the rail.

`letter-spacing` touches the horizontal spacing, which shall generally stay constant no matter what the default font size is